client/src/js/todos.js

```diff
@@ -0,0 +1,52 @@
+import { $ } from './util';
+import { createCardTemplate, createColumnTemplate } from './template';
+
+const getData = async (keyword) => {
+  const url = `http://localhost:3001/${keyword}`;
+  const res = await fetch(url);
+  const json = await res.json();
+  return json;
+};
+
+export const renderColumns = async () => {
+  const columnData = await getData('columns');
+  columnData.forEach((title) => {
+    $('main').insertAdjacentHTML('beforeend', createColumnTemplate(title));
+  });
+};
+
+const addCardHandler = (e) => {
+  const columnList = e.target.closest('.column').querySelector('.column-list');
+  !columnList.querySelector('.active')
+    ? columnList.insertAdjacentHTML('afterbegin', createCardTemplate())
+    : columnList.firstElementChild.remove();
```

Comment:
삼항연산자에서 부정값을 제거하고 순서를 바꿔두면 가독성이 올라갈 것 같네요~
```suggestion
  columnList.querySelector('.active')
    ? columnList.firstElementChild.remove()
    : columnList.insertAdjacentHTML('afterbegin', createCardTemplate());
```